soc/infineon/cat1b/psc3/Kconfig

```diff
@@ -0,0 +1,32 @@
+# copyright (c) (2016-2025), Cypress Semiconductor Corporation
+# (an Infineon company) or an affiliate of Cypress Semiconductor Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DT_COMPAT_ARM_CORTEX_M33 := arm,cortex-m33
+
+config SOC_SERIES_PSC3
+	select ARM
+	select CPU_CORTEX_M33 if $(dt_compat_enabled,$(DT_COMPAT_ARM_CORTEX_M33))
```

Comment:
this is not right, use cpu clusters properly in soc.yml then have the correct secondary symbol here which would be based on the soc e.g. `config SOC_PSC3M5FDS2ABQ1_M33` select the `CPU_CORTEX_M33` symbol